documentation/api/definitions.md

```diff
@@ -50,6 +87,17 @@ The `uid` property is useful for uniquely identifying a process when it is runni
 }
 ```
 
+## TraceProfile
+
+Enumeration that describes the type of diagnostic trace to capture. Each profile represents a list of event providers, event levels, and keywords.
+
+| Name | Description |
+|---|---|
+| `Cpu` | Tracks CPU usage and general .NET runtime information. |
+| `Http` | Tracks ASP[]().NET request handling and HttpClient requests. |
+| `Logs` | Tracks log events emitted at the `Debug` [log level](https://docs.microsoft.com/dotnet/api/system.diagnostics.tracing.eventlevel) or higher. |
```

Comment:
Is this debug or higher? Or does this default to UseAppFilters?

Reply:
I had to double check it, but the /trace?profile=logs call just uses the Debug level.

This is probably something we want to change and allow it to be overridden with a logLevel query parameter.

Reply:
Created #183 to track this